ember-cli-build.js

```diff
@@ -1,18 +1,32 @@
-/*jshint node:true*/
-/* global require, module */
-var EmberAddon = require('ember-cli/lib/broccoli/ember-addon');
-
-module.exports = function(defaults) {
-  var app = new EmberAddon(defaults, {
-    // Add options here
-  });
-
-  /*
-    This build file specifes the options for the dummy test app of this
-    addon, located in `/tests/dummy`
-    This build file does *not* influence how the addon or the app using it
-    behave. You most likely want to be modifying `./index.js` or app's build file
-  */
-
-  return app.toTree();
-};
+/*jshint node:true*/
+/* global require, module */
+var EmberAddon = require('ember-cli/lib/broccoli/ember-addon');
+
+module.exports = function(defaults) {
+  var options;
+
+  switch (process.env.TEST_SCENARIO) {
```

Comment:
Does this affect builds in consuming apps? I wonder if this code is best set in `.travis.yml` or something similar. For example, using a matrix for Travis tests: https://docs.travis-ci.com/user/environment-variables/